data/scripts.js

```diff
@@ -3579,4 +3579,225 @@ exports.BattleScripts = {
 
 		return pokemon;
 	},
+	randomBSSFactorySets: require('./bss-factory-sets.json'),
+	randomBSSFactorySet: function (template, slot, teamData, tier) {
+		let speciesId = toId(template.species);
+		// let flags = this.randomBSSFactorySets[tier][speciesId].flags;
+		let setList = this.randomBSSFactorySets[tier][speciesId].sets;
+		let effectivePool, priorityPool;
+		// arificially replicate Item Clause
+		let itemsMax = {"aguavberry":1, "alakazite":1, "assaultvest":1, "beedrillite":1, "blacksludge":1, "blastoisinite":1, "charizarditex":1, "chestoberry":1, "choiceband":1, "choicescarf":1, "choicespecs":1, "eeviumz":1, "ejectbutton":1, "electricmemory":1, "electriumz":1, "eviolite":1, "fightingmemory":1, "figyberry":1, "firiumz":1, "flameorb":1, "flyiniumz":1, "focussash":1, "gengarite":1, "ghostiumz":1, "groundmemory":1, "heatrock":1, "heracronite":1, "iapapaberry":1, "kangaskhanite":1, "keeberry":1, "leftovers":1, "lifeorb":1, "lightclay":1, "lucarionite":1, "mawilite":1, "medichamite":1, "metagrossite":1, "normaliumz":1, "occaberry":1, "pidgeotite":1, "pinsirite":1, "primariumz":1, "redcard":1, "rockiumz":1, "rockyhelmet":1, "salamencite":1, "scizorite":1, "sharpedonite":1, "sitrusberry":1, "slowbronite":1, "smoothrock":1, "steelmemory":1, "steelixite":1, "thickclub":1, "toxicorb":1, "venusaurite":1, "wateriumz":1, "weaknesspolicy":1};
```

Comment:
The reason that I limited the items here was because the Item Clause included in Standard GBU (in ``/data/rulesets,js``) only applies to the teambuilder validator (from what I can tell) and during testing I was able to generate teams with multiple of the same item, going against the Item Clause that BSS has. I was unaware of the inconsistency with the 8 missing items though, I'll rectify that when I next update the sets.